packages/patternfly-4/react-core/README.md

```diff
@@ -49,64 +36,18 @@ export default <Button variant="primary">Button</Button>;
 
 All css related to each component is provided within it. There is no component level CSS to import.
 
-# Applying Red Hat Fonts (Optional) 
-If you would like to use Red Hat fonts instead of Overpass, simply add the class `.pf-m-redhat-font` to an element that wraps your application (ideally `<html>` or `<body>`) to adopt the CSS changes that introduce the Red Hat fonts into PatternFly.
-
-
-# Documentation
-
-This project uses Gatsby. For an overview of the project structure please refer to the [Gatsby documentation - Building with Components](https://www.gatsbyjs.org/docs/building-with-components/).
-
-A comprehensive list of components and detailed usage of each can be found on the [PatternFly React Docs][docs] website
-You can also find how each component is meant to be used from a design perspective on the [PatternFly 4 Core][patternfly-4] website.
-
-Note: All commands below assume you are on the root directory in this repository.
-
-### Install
+## Building
 
-Run to install all the dependencies
+To build just react-core, run `yarn run lerna run build --scope=@patternfly/react-core`.
 
-```sh
-yarn install && yarn build
-```
-
-### Running
```

Comment:
How do they know how to run it?

Reply:
We don't "run" `react-core`. We only build it. We should add to the root README.md `yarn start:pf3` and `yarn start:pf4` runs the separate environments.

Reply:
Yeah, but we should probably have steps here someone interested in seeing the demo's for react-core, so they can easily start the demo without having to dig for instructions how.